### PR TITLE
feat: Add export-service schema

### PR DESCRIPTION
**Pull request by CodyWMitchell**

### CODE REVIEW

File: schemas/apps/export-service/export-request.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{
```
Comment (josejulio) [inline schemas/apps/export-service/export-request.json:1]: Could you move this file to a `v1` folder?

Comment (josejulio) [reply schemas/apps/export-service/export-request.json:1]: i.e. to `schemas/apps/export-service/v1/export-request.json`

File: schemas/apps/export-service/export-request.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    ],
+    "additionalProperties": false,
+    "definitions": {
+      "ExportRequest": {
+        "description": "A request for data to be exported",
+        "type": "object",
+        "properties": {
+          "uuid": {
+            "description": "A unique identifier for the request",
+            "type": "string"
```
Comment (MikelAlejoBR) [inline schemas/apps/export-service/export-request.json]: I would add a `"format": "uuid"` here to enforce the UUID format.

File: schemas/apps/export-service/export-request.json [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+          "application",
+          "format",
+          "resource",
+          "x-rh-identity"
+        ],
+        "additionalProperties": false
+      }
+    }
+  }
+
```
Comment (MikelAlejoBR) [inline schemas/apps/export-service/export-request.json]: New line missing!

File: schemas/apps/export-service/export-request.json [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+              "json"
+            ]
+          },
+          "resource": {
+            "description": "The resource to be exported",
+            "type": "string"
+          },
+          "x-rh-identity": {
+            "description": "The identity header of the user making the request",
+            "type": "string"
```
Comment (MikelAlejoBR) [inline schemas/apps/export-service/export-request.json]: What do you think about adding a `"contentEncoding": "base64"` field to this object? Reference: https://json-schema.org/understanding-json-schema/reference/non_json_data.html#id1

**Review by MikelAlejoBR — APPROVED** [review]
Looking good!

File: schemas/apps/export-service/v1/export-request.json [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+              "json"
+            ]
+          },
+          "resource": {
+            "description": "The resource to be exported",
+            "type": "string"
+          },
+          "x-rh-identity": {
+            "description": "The identity header of the user making the request",
+            "type": "string"
```
Comment (josejulio) [inline schemas/apps/export-service/v1/export-request.json:45]: Could you add that this is expected as a `base64` string?

File: schemas/apps/export-service/v1/export-request.json [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+          },
+          "filters": {
+            "description": "The filters to be applied to the data",
+            "type": "object"
```
Comment (josejulio) [inline schemas/apps/export-service/v1/export-request.json:49]: Is this like a map of strings? or is a free-form object?

If it's the former, we can specify that all the properties are of type string by using:
```
"additionalProperties": { "type": "string" }
```

Comment (CodyWMitchell) [reply schemas/apps/export-service/v1/export-request.json:49]: `filters` is a free-form object.

File: schemas/apps/export-service/export-request.json (hunk 4 above)
Comment (CodyWMitchell) [reply schemas/apps/export-service/export-request.json]: Good idea!

**Review by MikelAlejoBR — APPROVED** [review]
LGTM! Nice!

**Review by josejulio — APPROVED** [review]
lgtm

**Pull request merged**